code/modules/paperwork/paper.dm

```diff
@@ -97,6 +97,33 @@
 	return
 
 /obj/item/weapon/paper/attack_self(mob/living/user as mob)
+	if(ishuman(user)) //apparently other races don't apply lipstick.
+		var/mob/living/carbon/human/H = user
+		if(!H.check_body_part_coverage(MOUTH))
+			if((H.attack_type == ATTACK_BITE) && (H.a_intent == I_HELP)) //if biting and helping
+				add_fingerprint(H)
+				if(H.lip_style)
+					to_chat(user, "<span class='notice'>You kiss the piece of paper, leaving a pair of lipstick lips.</span>")
```

Comment:
`a pair of lipstick lips` - this could use rewriting, sounds weird
`leaving a lipstick impression` or something